src/main/java/org/adridadou/ethereum/propeller/RpcEthereumFacadeProvider.java

```diff
@@ -46,27 +58,41 @@ public static InfuraBuilder forInfura(final InfuraKey key, EthereumRpcConfig con
 
     public static class InfuraBuilder {
         private final InfuraKey key;
-        private final EthereumRpcConfig config;
```

Comment:
why did you remove the final here ?

Reply:
Made a fallback to polling if the websocketService.connect() throws a ConnectException, ugly?

Reply:
ooh I see. No I think it's ok. and to come back to my question, did you try it against infura and it works fine ?